rustfmt-core/rustfmt-lib/src/formatting/syntux/session/ignore_path.rs

```diff
@@ -8,7 +10,11 @@ pub(crate) struct IgnorePathSet {
 
 impl IgnorePathSet {
     pub(crate) fn from_ignore_list(ignore_list: &IgnoreList) -> Result<Self, ignore::Error> {
-        let mut ignore_builder = GitignoreBuilder::new(ignore_list.rustfmt_toml_path());
+        let root = ignore_list
+            .rustfmt_toml_path()
+            .parent()
```

Comment:
why the parent here?

Reply:
Looking at [this CI failure](https://travis-ci.com/github/rust-lang/rustfmt/jobs/339857177), I noticed that we are ought to pass a directory to [`GitignoreBuilder::new`](https://docs.rs/ignore/0.4.15/ignore/gitignore/struct.GitignoreBuilder.html#method.new), hence this change.